.github/workflows/json_validation_deploying_biovalidator.yml

```diff
@@ -0,0 +1,91 @@
+---
```

Comment:
i think this is out of scope of this repo

Reply:
Sorry can you elaborate on this one, Alegria? Out of the scope of the repo the whole YML file, the GH action itself?

Reply:
I think testing if the current "main" branch of Biovalidator can be deployed as the instance is out of the scope of this repo. That test should be in the Biovalidator repo itself.

Reply:
Even if we don't need it here, giving the schemas is done (at least with how Biovalidator works for now) at the deployment level. We **need** to deploy it and give the references ``-r`` with the branch's schemas. 
The most we could perhaps do is have an installation in some GH space and deploy it with different schemas each time the GH actions are executed. But I feel like that's a hassle and would be just easier to install it at every push. 

For normal PR I tend to push only after a few commits are added to it, so I wouldn't say the GH action would be executed that many times. Besides, it normally takes less than 30 seconds (~20-25 secs I think).